src/Util/RegexHelper.php

```diff
@@ -47,7 +47,7 @@ final class RegexHelper
     public const PARTIAL_SINGLEQUOTEDVALUE     = '\'[^\']*\'';
     public const PARTIAL_DOUBLEQUOTEDVALUE     = '"[^"]*"';
     public const PARTIAL_ATTRIBUTEVALUE        = '(?:' . self::PARTIAL_UNQUOTEDVALUE . '|' . self::PARTIAL_SINGLEQUOTEDVALUE . '|' . self::PARTIAL_DOUBLEQUOTEDVALUE . ')';
-    public const PARTIAL_ATTRIBUTEVALUESPEC    = '(?:' . '\s*=' . '\s*' . self::PARTIAL_ATTRIBUTEVALUE . ')';
+    public const PARTIAL_ATTRIBUTEVALUESPEC    = '(?:' . '\s*=?' . '\s*' . self::PARTIAL_ATTRIBUTEVALUE . '?)';
```

Comment:
Let's keep this as-is.  But, when we reference this fragment, we can add `?` to the end to make this entire group optional.